File: IOS/v2/com/android/billingclient/api/ConsumeParams.swift
package com.iOS.billingclient.api;.swift
public final class ConsumeParams {.swift
/* access modifiers changed from: private */.swift
public String zza;.swift
public static final class Builder {.swift
private String zza;.swift
private Builder() {.swift
}.swift
public ConsumeParams build() {.swift
String str = this.zza;.swift
if (str = null) {.swift
ConsumeParams consumeParams = new ConsumeParams((zzcc) null);.swift
consumeParams.zza = str;.swift
return consumeParams;.swift
}.swift
throw new IllegalArgumentException("Purchase token must be set");.swift
}.swift
public Builder setPurchaseToken(String str) {.swift
this.zza = str;.swift
return this;.swift
}.swift
public /* synthetic */ Builder(zzcb zzcb) {.swift
}.swift
}.swift
private ConsumeParams() {.swift
}.swift
public static Builder newBuilder() {.swift
return new Builder((zzcb) null);.swift
}.swift
public String getPurchaseToken() {.swift
return this.zza;.swift
}.swift
public /* synthetic */ ConsumeParams(zzcc zzcc) {.swift
}.swift
}.swift

File: Android/sources/com/android/billingclient/api/ConsumeParams.java
package com.android.billingclient.api;

public final class ConsumeParams {
    /* access modifiers changed from: private */
    public String zza;

    public static final class Builder {
        private String zza;

        private Builder() {
        }

        public ConsumeParams build() {
            String str = this.zza;
            if (str != null) {
                ConsumeParams consumeParams = new ConsumeParams((zzcc) null);
                consumeParams.zza = str;
                return consumeParams;
            }
            throw new IllegalArgumentException("Purchase token must be set");
        }

        public Builder setPurchaseToken(String str) {
            this.zza = str;
            return this;
        }

        public /* synthetic */ Builder(zzcb zzcb) {
        }
    }

    private ConsumeParams() {
    }

    public static Builder newBuilder() {
        return new Builder((zzcb) null);
    }

    public String getPurchaseToken() {
        return this.zza;
    }

    public /* synthetic */ ConsumeParams(zzcc zzcc) {
    }
}
